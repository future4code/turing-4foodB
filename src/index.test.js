import Index from './index.js';

it('Renderiza sem crashar', () => {
  expect(
    JSON.stringify(
      Object.assign({}, Index, { _reactInternalInstance: 'testingAppRender' }),
    ),
  ).toMatchSnapshot();
});