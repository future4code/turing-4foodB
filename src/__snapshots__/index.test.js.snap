// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderiza sem crashar 1`] = `"{\\"_reactInternalInstance\\":\\"testingAppRender\\"}"`;
